test/ios/mocked_data/test_get_network_instances/normal/show_ip_interface_brief.txt

```diff
@@ -1,17 +1,17 @@
-Load for five secs: 0%/0%; one minute: 1%; five minutes: 1%
```

Comment:
I'd ask you to refrain from changing the existing test-cases, but just add new ones (as you already did). In other words, please keep the `no_vrf` case and revert the changes under `normal`. Of course, if there's anything incorrect with the existing data, please clarify here.

Reply:
Sure.  I just re-read the PR description and the test data I threw together was irrelevant anyway.   New test cases incoming.

Reply:
Relevant test cases added and I dirty reverted the originals by copying them and pushing them again with https://github.com/napalm-automation/napalm/pull/1095/commits/4dff67c5364c7db2f5240e7af802dc76bd6b8902.  Hopefully that's not a problem.

Reply:
made a dog's breakfast out of that.  The normal test case is now failing in a way I don't understand:

    AssertionError: Expected result varies on some keys 
    
    {
        "default": {
            "interfaces": {
                "interface": {
                    "GigabitEthernet0": {
                        "result": null,
                        "expected": {}
                    }
                }
            }
        },
        "Mgmt-intf": {
            "interfaces": {
                "interface": {
                    "Gi0": {
                        "result": null,
                        "expected": {}
                    },
                    "GigabitEthernet0": {
                        "result": {},
                        "expected": null
                    }
                }
            }
        }
    }

and the data is actually invalid.  GigabitEthernet0 appears in both default and Mgmt-intf instances which this PR corrects.

Reply:
No problem at all, I don't mind what method you use as long as we get to there ;)
Looks like the tests are failing on the `normal` test case now, I think you can just copy the content from https://github.com/napalm-automation/napalm/tree/develop/test/ios/mocked_data/test_get_network_instances/normal and should be fine.

Reply:
The tests were failing with https://github.com/napalm-automation/napalm/tree/develop/test/ios/mocked_data/test_get_network_instances/normal for some reason and the data was invalid anyway because [Gi0 in Mgmt-intf](https://github.com/napalm-automation/napalm/blob/develop/test/ios/mocked_data/test_get_network_instances/normal/expected_result.json#L55) unwraps to GigabitEthernet0 which also appears in the [default instance](https://github.com/napalm-automation/napalm/blob/develop/test/ios/mocked_data/test_get_network_instances/normal/expected_result.json#L32).  Replaced the test case with my new one.

Reply:
Okay, makes sense